src/JJMasterData.Core/DataDictionary/IconType.cs

```diff
@@ -2,698 +2,697 @@
-    SortNumericAsc = 551,
-    SortNumericDesc = 552,
-    SortUp = 553,
-    Soundcloud = 554,
-    SpaceShuttle = 555,
-    Spinner = 556,
-    Spoon = 557,
-    Spotify = 558,
-    Square = 559,
-    SquareO = 560,
-    StackExchange = 561,
-    StackOverflow = 562,
-    Star = 563,
-    StarHalf = 564,
-    StarHalfEmpty = 565,
-    StarHalfFull = 566,
-    StarHalfO = 567,
-    StarO = 568,
-    Steam = 569,
-    SteamSquare = 570,
-    StepBackward = 571,
-    StepForward = 572,
-    Stethoscope = 573,
-    StickyNote = 574,
-    StickyNoteO = 575,
-    Stop = 576,
-    StopCircle = 577,
-    StopCircleO = 578,
-    StreetView = 579,
-    Strikethrough = 580,
-    Stumbleupon = 581,
-    StumbleuponCircle = 582,
-    Subscript = 583,
-    Subway = 584,
-    Suitcase = 585,
-    SunO = 586,
-    Superscript = 587,
-    Support = 588,
-    Table = 589,
-    Tablet = 590,
-    Tachometer = 591,
-    Tag = 592,
-    Tags = 593,
-    Tasks = 594,
-    Taxi = 595,
-    Television = 596,
-    TencentWeibo = 597,
-    Terminal = 598,
-    TextHeight = 599,
-    TextWidth = 600,
-    Th = 601,
-    ThLarge = 602,
-    ThList = 603,
-    ThumbTack = 604,
-    ThumbsDown = 605,
-    ThumbsODown = 606,
-    ThumbsOUp = 607,
-    ThumbsUp = 608,
-    Ticket = 609,
-    Times = 610,
-    TimesCircle = 611,
-    TimesCircleO = 612,
-    Tint = 613,
-    ToggleDown = 614,
-    ToggleLeft = 615,
-    ToggleOff = 616,
-    ToggleOn = 617,
-    ToggleRight = 618,
-    ToggleUp = 619,
-    Trademark = 620,
-    Train = 621,
-    Transgender = 622,
-    TransgenderAlt = 623,
-    Trash = 624,
-    TrashO = 625,
-    Tree = 626,
-    Trello = 627,
-    Tripadvisor = 628,
-    Trophy = 629,
-    Truck = 630,
-    Try = 631,
-    TTY = 632,
-    Tumblr = 633,
-    TumblrSquare = 634,
-    TurkishLira = 635,
-    Tv = 636,
-    Twitch = 637,
-    Twitter = 638,
-    TwitterSquare = 639,
-    Umbrella = 640,
-    Underline = 641,
-    Undo = 642,
-    University = 643,
-    Unlink = 644,
-    Unlock = 645,
-    UnlockAlt = 646,
-    Unsorted = 647,
-    Upload = 648,
-    Usb = 649,
-    Usd = 650,
-    User = 651,
-    UserMd = 652,
-    UserPlus = 653,
-    UserSecret = 654,
-    UserTimes = 655,
-    Users = 656,
-    Venus = 657,
-    VenusDouble = 658,
-    VenusMars = 659,
-    Viacoin = 660,
-    VideoCamera = 661,
-    Vimeo = 662,
-    VimeoSquare = 663,
-    Vine = 664,
-    Vk = 665,
-    VolumeDown = 666,
-    VolumeOff = 667,
-    VolumeUp = 668,
-    Warning = 669,
-    Wechat = 670,
-    Weibo = 671,
-    Weixin = 672,
-    Whatsapp = 673,
-    Wheelchair = 674,
-    WIFI = 675,
-    WikipediaW = 676,
-    Windows = 677,
-    Won = 678,
-    Wordpress = 679,
-    Wrench = 680,
-    Xing = 681,
-    XingSquare = 682,
-    YCombinator = 683,
-    Y_COMBINATOR_SQUARE = 684,
-    Yahoo = 685,
-    Yc = 686,
-    YcSquare = 687,
-    Yelp = 688,
-    YEN = 689,
-    Youtube = 690,
-    YoutubePlay = 691,
-    YoutubeSquare = 692
-
+  Adjust = '\uf042',
```

Comment:
Why didn't I do this before? 
I believed couldn't assign char in an enum. I feel like a monkey

Reply:
Me too, I discovered on the Discord server code review, it's because the `char` translates to `int`

Reply:
What happens when you serialize?

Reply:
this causes a breaking change in the mobile too.